docs/sql_cheatsheet.md

```diff
@@ -57,36 +65,38 @@ SELECT * FROM trades;
 
 ## Fix trade still open after a /forcesell
 
+Note: This should not be necessary, as forcesell orders are closed automatically by the bot on the next iteration.
```

Comment:
Maybe we should change the title of this section to "fix trade still open after manual sell on exchange" or something similar, since that is the more common usecase for this query.